portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -10,9 +13,14 @@
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
+    private final Gson gson = new Gson();
+    private final ArrayList<String> message = new ArrayList<String>(
+        Arrays.asList("Welcome To Fortress Inquisitorius!",
+        "It's a beautiful day!", "Have a Starsome Day!"));
```

Comment:
When working with long lists add each item on a new line for improved readability.
```suggestion
        Arrays.asList("Welcome To Fortress Inquisitorius!",
        "It's a beautiful day!",
        "Have a Starsome Day!"
        ));
```

Reply:
Noted!